init-interfaces.sh

```diff
@@ -87,7 +87,7 @@ for dev in $(nmcli device status | awk '/ethernet/ {print $1}'); do
       ;;
     *)
       ;;
-   esac
+  esac
```

Comment:
please move this to the commit I suggested earlier. the update of the expected secondary nic state is the only change that should be here